cmd/krew/cmd/upgrade.go

```diff
@@ -53,14 +54,18 @@ kubectl plugin upgrade foo bar"`,
 			pluginNames = args
 		}
 
+		executedKrewVersion, _, err := environment.GetExecutedVersion(paths.InstallPath(), os.Args[0], environment.Realpath)
```

Comment:
args[0] is not the absolute binary location. We need `os.Executeable()`

Reply:
> args[0] is not the absolute binary

This worked fine when I manually tested. But I'll fix.